RELEASES.md

```diff
@@ -1,5 +1,10 @@
 # Unreleased
 
+# 0.9.0
+## Changed
+- Smaller stack usage during final exponentiation
+- Serialization of Gt elements
```

Comment:
From the text it sounds to me like we implemented the serialization recently, but I think it's about the change in the endianness?

Reply:
So mainstream changes?

Reply:
All of this file was just from the upstream. I added some notes providing the changes in this fork

Reply:
thanks!